eHealthAfrica/aether
# feat: "identity mapping"

This PR adds an "Add Identity Mapping" button, which, when clicked, generates a set of default entity types and mappings.

How to test: 
- Open the ui, paste some json into the "JSON data" section
- Click "Derive Schema"
- Click "Add Identity Mapping"
- Check that the created entity types and mappings were created and that they conform to your schema.
